ocaml/xapi/helpers.ml

```diff
@@ -1736,6 +1736,12 @@ end = struct
           ; "TIMEOUTclose = 1"
           ; "options = CIPHER_SERVER_PREFERENCE"
           ; "sslVersion = TLSv1.2"
+          ; ""
+          ; "# xapi connections use SNI 'pool' to request a cert"
+          ; "[pool]"
+          ; "connect = 80"
+          ; "sni = xapi:pool"
+          ; Printf.sprintf "cert = %s" cert
```

Comment:
If I understand correctly, this shouldn't break master, because if a client does not specify an SNI parameter (which should always be the case right now) settings inside the [xapi] section will be used

However don't we need the [pool] section to have the same properties as above, for example the ciphers or the tls version?

Reply:
No. The `xapi` in `xapi:pool` refers to the `[xapi]` section for parameters.

Reply:
Ah I see. We presumably specify `connect` a second time, because the alternative 'sni cert' may be the certificate for an HTTP server running on a different port (however in our case it's the same server)